src/main/java/com/example/demo/model/Address.java

```diff
@@ -0,0 +1,136 @@
+package com.example.demo.model;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@Entity
+public class Address implements Serializable{
+	
+	@Id
+	@GeneratedValue(strategy= GenerationType.AUTO)
+	private int addressId; 
+	
+	private String city;
+	private String state;
+	private int pincode;
+	
+	@OneToOne(targetEntity=User.class)
+	@MapsId
```

Comment:
Use of @MapsId?

Reply:
Use to map the primary key ... its working fine with so Didn't removed it